frontend/test/__support__/mocks.js

```diff
@@ -6,6 +6,8 @@ global.window.matchMedia = () => ({
   removeEventListener: () => {},
 });
 
+HTMLElement.prototype.scrollIntoView = () => {}; // used under the hood in Mantine's Select component
```

Comment:
Doesn't need to happen in this PR, but we do have a `mockScrollIntoView` function in `_support_/ui.tsx`. Maybe we should just have it mocked by default in unit tests since so many Mantine components expect it to be there now.

Reply:
> Maybe we should just have it mocked by default in unit tests since so many Mantine components expect it to be there now.

Good idea, I'll submit a follow-up PR :+1: